pyproject.toml

```diff
@@ -19,6 +19,7 @@ urllib3 = "1.26.9"
 gen3 = "^4.11.4"
 jsonobject = "^2.0.0"
 pyarrow = "^8.0.0"
+paramiko = "^3.5.1"
```

Comment:
The repo structure is confusing and it's hard to understand what all the scripts are for. I don't know what this pyproject.toml is used for, but let's split this script and its requirements from the rest of the repo. Instead of the `etl` directory let's create a new directory for this script with a new pyproject.toml, and ideally a readme.

Eventually we should add a GH workflow to push the lambda function automatically.

Reply:
That sounds like a great idea, I can make that change.